### ISSUE TITLE
NullPointerException: Cannot get property 'parameters' on null object  when contract urlPath does not have any parameters

### ISSUE THREAD

**Issue by tdziurko** [Codearte/accurest#146]
I am getting following stacktrace on a simple project using accurest

```
Caused by: java.lang.NullPointerException: Cannot get property 'parameters' on null object
    at io.codearte.accurest.builder.MockMvcSpockMethodBodyBuilder.buildUrlFromUrlPath(MockMvcSpockMethodBodyBuilder.groovy:77)
    at io.codearte.accurest.builder.MockMvcSpockMethodBodyBuilder.buildUrl(MockMvcSpockMethodBodyBuilder.groovy:71)
    at io.codearte.accurest.builder.MockMvcSpockMethodBodyBuilder.when(MockMvcSpockMethodBodyBuilder.groovy:37)
    at io.codearte.accurest.builder.SpockMethodBodyBuilder.whenBlock(SpockMethodBodyBuilder.groovy:50)
    at io.codearte.accurest.builder.SpockMethodBodyBuilder.appendTo(SpockMethodBodyBuilder.groovy:34)
    at io.codearte.accurest.builder.MethodBuilder.appendTo(MethodBuilder.groovy:39)
    at io.codearte.accurest.builder.BlockBuilder.addBlock(BlockBuilder.groovy:59)
    at io.codearte.accurest.builder.ClassBuilder$_build_closure4.doCall(ClassBuilder.groovy:104)
    at io.codearte.accurest.builder.ClassBuilder.build(ClassBuilder.groovy:103)
    at io.codearte.accurest.builder.ClassBuilder$build$4.call(Unknown Source)
    at io.codearte.accurest.SingleTestGenerator.buildClass(SingleTestGenerator.groovy:63)
```

**Comment by tdziurko** [comment]
Workaround for this issue is to use empty queryParameters

```
urlPath('/available-channels') {
        queryParameters {}
}
```

**Comment by jkubrynski** [comment]
Fixed in 0.9.2
